defaults/main.yml

```diff
@@ -4,3 +4,6 @@ server:
 php:
     ppa: "php5-5.6"
     packages: ["php5-cli", "php5-intl", "php5-mcrypt"]
+
+php_version: "{% if php.ppa == 'php-7.0' %}php7.0{% else %}php5{% endif %}"
+php_config_prefix: "{% if php.ppa == 'php-7.0' %}/etc/php/7.0{% else %}/etc/php5{% endif %}"
```

Comment:
For the directories I wasn't sure if that was the default ot not. So I'm fine with that solution.
Same goes for the conditionals for the time being. We can always improve later on if needed and if we find a better way.